src/js/components/controls/ButtonSet.jsx

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from 'react'
+import { ButtonGroup, Button } from 'semantic-ui-react'
+
+class ButtonSet extends Component {
```

Comment:
This component is designed specifically for settings, therefore it should be in settings directory. It should also be named accordingly.